script.js

```diff
@@ -1,4 +1,1250 @@
+
+  var myOutputValue = errorMessage;
+
+  var judgement = function (playerHand, computerHand) {
+    if (playerHand == computerHand) {
+      return "draw";
+    }
+    if (playerHand == "scissors" && computerHand == "stone") {
+      return "lose";
+    }
+    if (playerHand == "scissors" && computerHand == "paper") {
+      return "win";
+    }
+    if (playerHand == "paper" && computerHand == "scissors") {
+      return "lose";
+    }
+    if (playerHand == "paper" && computerHand == "stone") {
+      return "win";
+    }
+    if (playerHand == "stone" && computerHand == "paper") {
+      return "lose";
+    }
+    if (playerHand == "stone" && computerHand == "scissors") {
+      return "win";
+    }
+  };
+
+  if (judgement(input, gameHand) == "draw") {
+    myOutputValue = draw;
+  }
+  if (judgement(input, gameHand) == "lose") {
+    myOutputValue = lose;
+  }
+  if (judgement(input, gameHand) == "win") {
+    myOutputValue = win;
+  }
+
+  return myOutputValue + endingMessage;
+}; */
+
+//============================================================================================================================================================================
+
+var Code02 = "PROJECT 1: SCISSORS PAPER STONE PART 1: REVERSED MODE";
+/* 
+var main = function (input) {
+  var gameHand = readGameHandNmbr(randomNumber());
+  var win = `The computer chose ${gameHand.toUpperCase()}.<br><br>You chose ${input.toUpperCase()}.<br><br> ***You WIN!***`;
+  var lose = `The computer chose ${gameHand.toUpperCase()}.<br><br>You chose ${input.toUpperCase()}.<br><br> ~You LOSE~`;
+  var draw = `The computer chose ${gameHand.toUpperCase()}.<br><br>You chose ${input.toUpperCase()}.<br><br> =It's a DRAW!=`;
+  var errorMessage = `xxxxxxxxx INPUT ERROR xxxxxxxxxx. <br><br> There are only 3 options: 'scissors', 'paper' or 'stone'.`;
+  var endingMessage = `<br><br> To play the next game, please type 'scissors', 'paper' or 'stone'. <br><br> '<i>If something doesn't look right, you could always look at its complement view.</i>`;
+
+  var myOutputValue = errorMessage;
+
+// Code Block for reversing
+  if (input.toLowerCase() == "reversed scissors") {
+    input = "paper";
+  }
+  if (input.toLowerCase() == "reversed paper") {
+    input = "stone";
+  }
+  if (input.toLowerCase() == "reversed stone") {
+    input = "scissors";
+  }
+
+//code Block for judging who wins
+  if (
+    (input.toLowerCase() == "scissors" && gameHand == "paper") ||
+    (input.toLowerCase() == "paper" && gameHand == "stone") ||
+    (input.toLowerCase() == "stone" && gameHand == "scissors")
+  ) {
+    myOutputValue = lose;
+  }
+
+  if (
+    (input.toLowerCase() == "scissors" && gameHand == "stone") ||
+    (input.toLowerCase() == "paper" && gameHand == "scissors") ||
+    (input.toLowerCase() == "stone" && gameHand == "paper")
+  ) {
+    myOutputValue = win;
+  }
+
+  if (input.toLowerCase() == gameHand) {
+    myOutputValue = draw;
+  }
+
+  return myOutputValue + endingMessage;
+};
```

Comment:
I would say that you could either assign input to be a new variable called reversedInput or just keep the reversed formatting and just put comments above the if statements to remind that this is reverse. This is because if someone looks at the block of code they wouldn't be able to tell that this is a reverse SPS if they didnt look at the part where you reverse their input. We generally want to avoid confusion in that regard even though it may be a struggle to code reverse logic easily - so naming input to be reversedInput in the if statements would be more accurate

 Also, the idea is that we generally want to add on functionalities to code rather than just create separate blocks of code for each particular functionality. Apologies since i didn't mention that.